chainer/links/activation/dropconnect.py

```diff
@@ -0,0 +1,83 @@
+import math
+
+from chainer import cuda
+from chainer.functions.noise import dropconnect
+from chainer import initializers
+from chainer import link
+
+
+class Dropconnect(link.Link):
+
+    """Linear layer using dropconnect.
+
+    Args:
+        in_size (int): Dimension of input vectors. If None, parameter
+            initialization will be deferred until the first forward data pass
+            at which time the size will be determined.
+        out_size (int): Dimension of output vectors.
+        wscale (float): Scaling factor of the weight matrix.
+        bias (float): Initial bias value.
+        nobias (bool): If ``True``, then this function does not use the bias.
+        initialW (2-D array): Initial weight value. If ``None``, then this
+            function uses to initialize ``wscale``.
+            May also be a callable that takes ``numpy.ndarray`` or
+            ``cupy.ndarray`` and edits its value.
+        initial_bias (1-D array): Initial bias value. If ``None``, then this
+            function uses to initialize ``bias``.
+            May also be a callable that takes ``numpy.ndarray`` or
+            ``cupy.ndarray`` and edits its value.
+
+    .. seealso:: :func:`~chainer.functions.dropconnect`
+
+    Attributes:
+        W (~chainer.Variable): Weight parameter.
+        b (~chainer.Variable): Bias parameter.
+
+    """
+
+    def __init__(self, in_size, out_size, wscale=1, bias=0, nobias=False,
+                 ratio=.5, initialW=None, initial_bias=None):
+        super(Dropconnect, self).__init__()
+
+        # For backward compatibility
+        self.initialW = initialW
+        self.wscale = wscale
+
+        self.out_size = out_size
+        self.ratio = ratio
+        # For backward compatibility, the scale of weights is proportional to
```

Comment:
`DropConnect` does not have backward compatibility to support. So this comment is not needed. As for implementation, I agree with this scale change because it is same behavior of `Convolution2D`